example/spec/syncGranule/SyncGranuleDuplicateHandlingErrorSpec.js

```diff
@@ -0,0 +1,129 @@
+const fs = require('fs');
+const path = require('path');
+const { Collection } = require('@cumulus/api/models');
+const {
+  constructCollectionId,
+  stringUtils: { globalReplace }
+} = require('@cumulus/common');
+const {
+  addCollections,
+  buildAndExecuteWorkflow,
+  LambdaStep,
+  deleteCollections,
+  listCollections
+} = require('@cumulus/integration-tests');
+
+const {
+  loadConfig,
+  timestampedTestDataPrefix,
+  deleteFolder,
+  uploadTestDataToBucket
+} = require('../helpers/testUtils');
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+
+const s3data = [
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf'
+];
+
+describe('The Sync Granules workflow is configured to handle duplicates as an error', () => {
+  const testDataFolder = timestampedTestDataPrefix(`${config.stackName}-SyncGranuleDuplicateHandlingError`);
+  const inputPayloadFilename = './spec/syncGranule/SyncGranuleDuplicateHandlingError.input.payload.json';
+  const collection = { name: 'MOD09GQ_duplicateHandlingError', version: '006' };
+  const provider = { id: 's3_provider' };
+  const catchTaskName = 'SyncGranuleCatchDuplicateErrorTest';
+  const taskName = 'SyncGranule';
+  const collectionsDirectory = './data/collections/syncGranule';
+  const fileStagingDir = 'custom-staging-dir';
+  let destFileDir;
+  let existingFileKey;
+
+  const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
+  // update test data filepaths
+  const updatedInputPayloadJson = globalReplace(inputPayloadJson, 'cumulus-test-data/pdrs', testDataFolder);
+  const inputPayload = JSON.parse(updatedInputPayloadJson);
```

Comment:
If you use `setupTestGranuleForIngest` from `../helpers/granuleUtils` it will generate a random granule Id for your test to reduce the chance of test collisions. It _will_ require you to upload the test data before you create the `inputPayload`, though.